src/validateInput.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const { EXPECTED_NUMBER_LENGTH } = require('./constants');
+
+function validateInput(input) {
+  const inputSplited = input.split('');
+  const inputSet = new Set(inputSplited);
+  const hasValidLength = inputSplited.length === EXPECTED_NUMBER_LENGTH;
+  const hasDistinctDigits = inputSplited.length === inputSet.size;
+
+  return hasValidLength && hasDistinctDigits;
+};
```

Comment:
Also you need to check if user entered only numbers
If user enter `123q` your game will never finish